packages/api/db/tables.sql

```diff
@@ -142,3 +142,13 @@ CREATE TABLE IF NOT EXISTS migration_event
     data        jsonb,
     inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
+
+-- URLs of backups of user uploads
+CREATE TABLE IF NOT EXISTS backup
+(
+    id          BIGSERIAL PRIMARY KEY,
+    upload_id   BIGINT NOT NULL REFERENCES public.upload (id),
+    url         TEXT NOT NULL,
+    inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
+    deleted_at  TIMESTAMP WITH TIME ZONE
```

Comment:
We don't track delete in web3.storage land. Curious on what you foresee on needing to add a deleted ts?

Reply:
Not sure, I thought I just copied this...